lib/wallaby/query.ex

```diff
@@ -168,6 +213,9 @@ defmodule Wallaby.Query do
     }
   end
 
+  @doc """
+  Looks for an option with the given text appears.
```

Comment:
I think here we can just say: "Looks for an option that contains the given text".